### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by subhanzakizada**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,6 +13,27 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
+    <!-- Qupte -->
+    <div class='quote-container' id='quote-container'>
+        <div class='quote-text'>
+            <i class='fas fa-quote-left'></i>
+            <span id='quote'>Don't judge each day by the harvest you reap but by the seeds that you plant.</span>
```
Comment (adilliadil) [inline index.html:20]: aren't the quotes autogenerated?

**Review by adilliadil — APPROVED** [review]
lgtm

File: index.html (hunk 1 above)
Comment (subhanzakizada) [reply index.html:20]: they are. in here i just wrote one to test the quote container and how it looks etc

Comment (adilliadil) [reply index.html:20]: sounds good then, don't forget to remove the test later on

**Pull request merged**